api/src/utils/fecthAPIGamesget.js

```diff
@@ -14,6 +14,8 @@ async function fecthAPIGames(url, count) {
 
 function datosTraidosDeLaApi(datos) {
   try {
+    // datos.genres.replace("Action", "hola")
```

Comment:
Remove unnecesary comment

Reply:
remove comment, this conversation not resolved